components/plurals/src/rules/raw_operands.rs

```diff
@@ -0,0 +1,48 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+use crate::PluralOperands;
+
+#[cfg(doc)]
+use crate::PluralRules;
+
+/// A struct for low-level users who want to construct a [`PluralOperands`]
```

Comment:
Nit: Add the Experimental boilerplate here, too. It should be on all experimental modules, structs, and enums